telco-hub/configuration/reference-crs-kube-compare/metadata.yaml

```diff
@@ -75,6 +75,47 @@ parts:
                 - templates
           - path: required/acm/thanosSecretPlacement.yaml
           - path: required/acm/thanosSecretPlacementBinding.yaml
+  - name: required-gitops
+    description: |-
+      https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/scalability_and_performance/telco-hub-ref-design-specs#telco-hub-gitops-operator-and-ztp-plugins_telco-hub
+    components:
+      - name: gitops-operator
+        allOf:
+          - path: required/gitops/addPluginsPolicyNS.yaml
+          - path: required/gitops/addPluginsMCSB.yaml
+          - path: required/gitops/addPluginsPolicy.yaml
+            config:
+              ignore-unspecified-fields: true
```

Comment:
Which fields are being added that make this necessary?

Reply:
Probably a leftover from testing. Deleted in the latest implementation.